blog/patterns.md

```diff
@@ -0,0 +1,153 @@
+
+<p align="center"><a href="https://cloudfour.com/thinks/mood-boards/"><img alt="Box Grow Example" src="https://res.cloudinary.com/cloudfour/image/upload/c_scale,f_auto,q_auto,w_400/v1580512879/blog-patterns/box-grow.png" /></a></p>
+
+The Grow modifier will add a [`u-textGrowX` class](https://cloudfour-patterns.netlify.com/patterns/utilities.html#text) using whatever size you pass in.
+
+```html
+[c4box grow=2]
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+[/c4box]
+```
+
+You can use `grow=0` to keep the text the same size as the rest of the post, which is useful for a sort of alert box.
+
+## Footnotes
+
+<p align="center"><a href="https://cloudfour.com/thinks/hey-hey-cloud-four-is-a-pwa/"><img alt="Footnotes Example" src="https://res.cloudinary.com/cloudfour/image/upload/c_scale,f_auto,q_auto,w_400/v1580512879/blog-patterns/footnotes.png" /></a></p>
+
+This is actually a feature of WordPress's version of [Markdown Extra](https://en.support.wordpress.com/markdown-quick-reference/).
+
+```md
+I have more [^1] to say up here.
+
+(other post content)
+
+[^1]: To say down here.
+```
+
+Footnotes marked up this way will be added to the bottom of the post with a link back to the original reference.
+
+## Figure Shortcode
+
+<p align="center"><a href="https://cloudfour.com/thinks/hey-hey-cloud-four-is-a-pwa/"><img alt="Figure Example" src="https://res.cloudinary.com/cloudfour/image/upload/c_scale,f_auto,q_auto,w_400/v1580512879/blog-patterns/figure.png" /></a></p>
+
+The Figure shortcode will create a [Figure](https://cloudfour-patterns.netlify.com/patterns/components/figure.html), with an optional caption and class.
+
+```html
+[c4figure caption="Our offline notification banner." class="example"]
+<img src="https://cloudfour.com/wp-content/uploads/2016/11/cloudfour-offline-banner-400.png" class="u-borderSm">
+[/c4figure]
+```
+
+Note the optional use of the `u-borderSm` utility class to add a light border to the image. Images with a white background may benefit from a border.
+
+## FlexEmbed Shortcode
+
+<p align="center"><a href="https://cloudfour.com/thinks/an-html-attribute-potentially-worth-4-4m-to-chipotle/"><img alt="Flex Embed Example" src="https://res.cloudinary.com/cloudfour/image/upload/c_scale,f_auto,q_auto,w_400/v1580512879/blog-patterns/flexembed.png" /></a></p>
+
+The FlexEmbed shortcode create a [Flex Embed](https://cloudfour-patterns.netlify.com/patterns/components/flex-embed.html), used to make an embed resize responsively while retaining its aspect ratio..
+
+```html
+[c4flexembed]
+<iframe width="420" height="315" src="https://www.youtube.com/embed/Nx64_N4AA04" frameborder="0" allowfullscreen></iframe>
+[/c4flexembed]
+```
+
+#### Ratio Modifier
+
+You can manually choose the aspect ratio of the embed by setting the `ratio` attribute to either `1by1, `2by1`, `3by1`, or `4by3`. You don't need to specify `16by9`, as that's the default.
```

Comment:
Missing a backtick

Reply:
Might be worth linking to the pattern, which is where supported ratios are defined: https://cloudfour-patterns.netlify.com/patterns/components/flex-embed.html

Reply:
There's a link to the pattern above. :)